compiler/aarch64/codegen/OMRRegisterDependencyStruct.hpp

```diff
@@ -53,6 +53,12 @@ struct RegisterDependencyExt: OMR::RegisterDependencyExt
 
    TR::RealRegister::RegNum getRealRegister() {return _realRegister;}
    TR::RealRegister::RegNum setRealRegister(TR::RealRegister::RegNum r) { return (_realRegister = r); }
+
+   /**
+    * @return Answers \c true if this register dependency records a register in
+    *         spill state; \c false otherwise.
+    */
+   bool isSpilledReg() { return _realRegister == TR::RealRegister::SpilledReg; }
```

Comment:
This looks the same on all platforms. Is there a possibility of commoning this up into a shared implementation?

Reply:
Yes.  I moved the `RealRegister` field up into the base class and shared by all architectures to allow this.